packages/autorest.typescript/CONTRIBUTING.md

```diff
@@ -0,0 +1,348 @@
+# Introduction  
+
+This package contains the High Level Client generation and the Rest Level Client generation from swagger. We will introduce both on how to contribute to high level client generation and rest level client generation.  
```

Comment:
For introduction, could you leverage and modify the sentense from https://github.com/Azure/autorest.typescript/blob/main/packages/autorest.typescript/README.md